lib/fizzy/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ add_library(fizzy::fizzy ALIAS fizzy)
 
 target_sources(
     fizzy PRIVATE
+    cxx20/bit.hpp
```

Comment:
Are we going to move `span` here too?

Reply:
Yes. We briefly discussed that on Telegram with @gumb0 some time ago.